docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -251,6 +251,10 @@ We need to map that to coala's convention as follows:
             normalize_column_numbers = True,
             ...)
 
+This is a reminder for developers that they need to do a proper research on
+linter bear, ensure the convention and ideally write tests to ensure
```

Comment:
Should be `bears`, no? :sweat_smile:

Reply:
i guess they should research particularly on `linter bear`

Reply:
To be more accurate, this should be `linter` instead of `linter bear`.